project/Build.scala

```diff
@@ -97,17 +97,22 @@ object Build {
   val publishedDottyVersion = referenceVersion
   val sbtDottyVersion = "0.5.5"
 
-  /** Version against which we check binary compatibility.
+  /** Minor version against which we check binary compatibility.
    *
-   *  This must be the latest published release in the same versioning line.
-   *  For example, if the next version is going to be 3.1.4, then this must be
-   *  set to 3.1.3. If it is going to be 3.1.0, it must be set to the latest
+   *  This must be the earliest published release in the same versioning line.
+   *  For example, if the next version is going to be 3.M.4, then this must be
+   *  set to 3.1.0. If it is going to be 3.1.0, it must be set to the earliest
    *  3.0.x release.
```

Comment:
This wording is confusing. Is the `M` a typo in the example? Should the second version be then specified as `3.M.0`? Isn't `the earliest 3.0.x` just `3.0.0`?

I suggest including explicit rules in this comment. Something along the lines of:
```
For a base version `3.M.P` the mimaPreviousDottyVersion should be set to:
* `3.M.0` if P > 0
* `3.(M-1).0` if P = 0
```

Reply:
You are right, it was meant as a `3.M.0`. I will use your version as it is much clearer and direct.